sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/UpdateResourceGroupOperation.cs

```diff
@@ -58,7 +58,7 @@ private ResourceGroup GetResourceGrouop(JsonDocument document)
         {
             var method = typeof(ResourceManager.Resources.Models.ResourceGroup).GetMethod("DeserializeResourceGroup", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Static);
             var obj = method.Invoke(null, new object[] { document.RootElement });
-            return new ResourceGroup(_operations, new ResourceGroupData(obj as ResourceManager.Resources.Models.ResourceGroup));
+            return new ResourceGroup(_operations, obj as ResourceGroupData);
```

Comment:
remove the reflection here and call the deserializer directly, can copy from Azure.ResourceManager.Resources class as an example